helm-charts/bytebase/templates/statefulset.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: bytebase
+  namespace: {{ template "bytebase.namespace" . }}
+spec:
+  selector:
+    matchLabels:
+      app: bytebase
+  serviceName: "bytebase"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: bytebase
+    spec:
+      containers:
+        - name: bytebase
+          image: "bytebase/bytebase:{{ .Chart.AppVersion }}"
```

Comment:
@suzaku does the case matter here? Since the field is "appVersion" instead of "AppVersion"

Reply:
Yes, the case matters and `AppVersion` is correct.

You can take a guess at why we have to use `.Chart` and `.AppVersion` 😁

Reply:
Seems like this could bite quite a few folks -_-||

Reply:
The authors of Helm are quite lazy, they use the `text/template` package in the Go standard library directly. So we are in fact writing Go here, and exported fields all start with a upper case letter.

Reply:
That explains.